mkdocs.yml

```diff
@@ -99,6 +98,7 @@ pages:
       - Application Configuration: 'developer-guide/local-config.md'
       - Configuring plugins: 'developer-guide/plugins-documentation.md'
       - Map Configuration: 'developer-guide/maps-configuration.md'
+      - How to change background list: 'developer-guide/how-to-change-background-list.md'
```

Comment:
From the conversation looks like you removed this file. If so, please remove also the entry in the menu.
```suggestion
```